.github/autosubmit.yml

```diff
@@ -0,0 +1,7 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file will be added to flutter's internal repository.
+# TODO: publish design doc and link here.
```

Comment:
It might be painful to go back and edit these all after the fact. I would recommend creating a wiki page for the time being, and pointing to it from here.

Reply:
Good call. Let me do that.

Reply:
cl for the doc link: https://critique.corp.google.com/cl/518693811

Reply:
Since this is an open source tool, corp isn't viable. Prefer setting something up in https://github.com/flutter/flutter/wiki